html/sub/demos/sup/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Пример с sup — sub — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
+  <style>
+    body {
+      background-color: #18191c;
+      color: white;
+      font-family: "Roboto Mono", monospace;
+    }
+
+    p {
+      font-size: 30px;
+    }
```

Comment:
```suggestion

    var {
      font-style: normal;
    }
```

На мой взгляд, курсив здесь только затрудняет чтение, и не способствует выразительности.